packages/forma-36-react-components/src/components/Note/Note.css

```diff
@@ -58,3 +65,19 @@
   display: flex;
   margin-top: 1px;
 }
+
+.Note__dismiss {
```

Comment:
I think the button needs some space on the left to avoid clashing with the text (image 1)

<img width="130" alt="Screenshot 2020-07-21 at 11 01 06" src="https://user-images.githubusercontent.com/7631029/88035007-c9f7e280-cb41-11ea-817d-7a4d6a6d0f06.png">

(I did it with `margin-left` but not sure if that's the right approach)

...and perhaps a `margin-bottom: auto` to avoid it having a taller click area than it should.

Before:
<img width="146" alt="Screenshot 2020-07-21 at 11 01 38" src="https://user-images.githubusercontent.com/7631029/88035072-e3009380-cb41-11ea-98a2-01c8b6a6f37e.png">

After:
<img width="205" alt="Screenshot 2020-07-21 at 11 02 05" src="https://user-images.githubusercontent.com/7631029/88035085-e8f67480-cb41-11ea-90df-285da185e66c.png">

Reply:
+1

Reply:
<img width="122" alt="Screenshot 2020-07-21 at 22 57 50" src="https://user-images.githubusercontent.com/156505/88106517-e4fc3e00-cba5-11ea-8593-7303b6da4125.png">

To keep the button floating right, I had to apply the margin to the middle content block (in case we show the close button). There might be better CSS solutions that I don't know off :)

Reply:
Looks good to me! 👍